features/support/cucumber_rails_gem_helper.rb

```diff
@@ -29,13 +30,21 @@ def add_cucumber_rails(options)
     end
   end
 
+  def add_sqlite3_gem
```

Comment:
Can we refactor this into the other method. In the other method, we should add selenium webdriver to be greater than 4.22 for latest rails and to not require webdrivers gem

Reply:
Sorry, I may not have understood this request. I've tried what I understood

Reply:
Now if you analyse the two methods `add_rails_conditional_gems` and `add_selenium_webdriver_gem` you'll see they're almost identical in branches, so they can be re-constituted back together

Reply:
> so they can be re-constituted back together

That's the first thing I've tried, but rubocop complained about complexity of the method, so I've splitted

If it is fine, I can disable rubocop on that method and merge the two methods back together

Reply:
Ah oki, it might be better I look into tidying this up a bit more later on then. Sorry. Thanks again!